### PR TITLE
improvements to final class and resources

### PR DESCRIPTION
**Pull request by SViccari**
Updates the slides for the final class. Simplifies the blog posts to accommodate time constraints.

To-Do:
- [x] Review and add to the list of Resources
- [x] Add content to slides that just have a header

final-project

### CODE REVIEW

File: class5.html [hunk 1]
```diff
@@ -83,548 +68,158 @@ <h2>Using the Inspector</h2>
...
-        <section>
-          <h2>Style the Blog Post Title</h2>
-          <p>Add a class to the blog post title.</p>
+          <h2>Style the Article Title (html & css)</h2>
+          <p>Add a class to the article title.</p>
+          <p>Style the article title to be larger and bold.</p>
 
           <pre>
             <code class="html" data-trim contenteditable>
               <p class="article-title">Best Florida Beaches</p>
```
Comment (edwardloveall) [inline class5.html]:
```suggestion
              <h2 class="article-title">Best Florida Beaches</h2>
```

Potentially another good place for semantic elements.

**Review by edwardloveall — COMMENTED** [review]
We could go much more semantic on this final project with `<article>` and `<h2>` and probably others. It's fine if we don't because of time or just cuz, frankly.

File: class5.html [hunk 2]
```diff
@@ -83,548 +68,158 @@ <h2>Using the Inspector</h2>
...
-          <p>Goal: Center the text.</p>
-
-          <p>Wrap the &lt;p&gt; and &lt;h1&gt; tags in a &lt;div&gt;.</p>
-
-          <pre>
-            <code class="css" data-trim contenteditable>
-              <div class="intro-section">
-                <p class="intro">Welcome!</p>
-                <h1 class="intro-about">Somewhere on a Beach</h1>
+              <div class="article">
```
[the remark this answers is not in the document]
Comment (SViccari) [reply class5.html]: yes! good catch 🎣

**Comment by SViccari** [comment]
Thank you @edwardloveall for the review 🐠 💛

**Pull request merged**